index.html

```diff
@@ -20,6 +20,11 @@
     <div class="cover">
       <header class="header">
         <h1 class="logo"><a href="#">My Logo</a></h1>
+        <ul class="nav-bar-links">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
```

Comment:
Very good job implementing a part of the suggestions of the previous reviewer. :clap: 
But still, some parts need to be addressed:

- [ ] Please try to match the colours of the nav links and of the logo with figma design. Keep in mind that you have to style the `a` tags. For that and you have some handy tools too as pseudo-classes like link, visited, hover and active. In your case, it's important to style at least the link and the visited state.
- [ ] it is not recommended and is not user friendly to make use of the horizontal scroll. Please, fix it!
![image](https://user-images.githubusercontent.com/83596135/154814263-cedc2c3f-cf89-4b0f-a7e2-f50366ab2ec0.png)


[OPTIONAL]
- [ ] Because you use fixed dimensions in your design, the UI is not fully responsive. 
This is what the design looks like with a screen of 990 px:
![image](https://user-images.githubusercontent.com/83596135/154814133-5d70c9ba-0ed6-4101-bb5f-a7fec0b38a57.png)

Reply:
Very good job implementing a part of the suggestions of the previous reviewer. :clap: 
But still, some parts need to be addressed:

- [ ] Please try to match the colours of the nav links and of the logo with figma design. Keep in mind that you have to style the `a` tags. For that and you have some handy tools too as pseudo-classes like link, visited, hover and active. In your case, it's important to style at least the link and the visited state.
- [ ] it is not recommended and is not user friendly to make use of the horizontal scroll. Please, fix it!
![image](https://user-images.githubusercontent.com/83596135/154814263-cedc2c3f-cf89-4b0f-a7e2-f50366ab2ec0.png)
hint: you can set in the mobile version, in the `body`, along with the other attributes, this one too: `overflow-x: hidden;`


[OPTIONAL]
- [ ] Because you use fixed dimensions in your design, the UI is not fully responsive. 
This is what the design looks like with a screen of 990 px, just before jumping to desktop view:
![image](https://user-images.githubusercontent.com/83596135/154814133-5d70c9ba-0ed6-4101-bb5f-a7fec0b38a57.png)